### PR TITLE
Update date to mention KubeCon NA 2022

### PR DESCRIPTION
**Pull request by amandapunch**
On the page https://kubernetes.io/ `Attend KubeCon North America on October 11-15, 2021` is still displayed even though KubeCon NA 2021 has passed. This PR updates the text to `Attend KubeCon North America on October 24-28, 2022`.

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
Resolves:
- #30718

**Comment by RinkiyaKeDad** [comment]
> Resolves:

Using "fixes" instead of "resolves" will link the two so that the issue closes automatically when the PR is merged :)

Fixes #30718

**Review by mehabhalodiya — APPROVED** [review]
Thank you @amandapunch for fixing!
/lgtm

**Comment by mehabhalodiya** [comment]
As said by @sftim [comment](https://github.com/kubernetes/website/issues/30718#issuecomment-985373995)   
I think there's no need to modify `website/content/ko/_index.html`.

File: content/en/_index.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Attend KubeCon North America on October 11-15, 2021</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Attend KubeCon North America on October 24-28, 2022</a>
```
Comment (sftim) [inline content/en/_index.html:46]: Please move this one to happen after KubeCon Europe 2022

**Comment by sftim** [comment]
@amandapunch this fix helps, but: we'd want a follow up fix right away to get the event order right.

Would you be willing to revise this to put the events in chronological order, and only cover English language changes?
If it's easier to do so, you're welcome to close this PR and open a new one that replaces it.

File: content/en/_index.html (hunk 1 above)
Comment (amandapunch) [reply content/en/_index.html]: Good catch - I just made another commit to fix this.

**Comment by amandapunch** [comment]
> As said by @sftim [comment](https://github.com/kubernetes/website/issues/30718#issuecomment-985373995) I think there's no need to modify `website/content/ko/_index.html`.

Thanks for letting me know - I reverted my changes to the Korean page

**Review by anubha-v-ardhan — COMMENTED** [review]
Seems all good now
/lgtm

**Comment by anubha-v-ardhan** [comment]
/remove-language/ko

**Comment by anubha-v-ardhan** [comment]
/remove-language/ko

**Pull request merged**